run_combination_test.py

```diff
@@ -71,6 +71,7 @@ def get_shuffle_params(params, index):
     if params['h5py'] == '2.6':
         conf['requires'].append('h5py<2.7')
     elif params['h5py'] == '2.5':
+        conf['requires'].append('numpy<1.10')
```

Comment:
Please add comment why we always need to install numpy before h5py.
"We need to install numpy *before* h5py because h5py uses numpy in its setup script."